services/api/handlers/users.js

```diff
@@ -1,137 +1,142 @@
 var boom = require('boom');
 
-module.exports = function(request, reply) {
-  if (request.method === 'get') {
-    return request.server.methods.db.findUser(
-      [request.params.user],
-      function(err, result) {
-        if ( err ) {
-          return reply(err);
-        }
+exports.get = function(request, reply) {
+  request.server.methods.db.findUser(
```

Comment:
Have you considered moving these server methods into the `pre` handlers for each route?